sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -17219,15 +17219,20 @@ namespace Azure.Storage.Blobs.Models
     /// </summary>
     public enum DeleteSnapshotsOption
```

Comment:
What about BlobAndSnapshot instead of IncludeSnapshots?

Reply:
I like `IncludeSnapshots` a little more because it matches the service concept and documentation.